### ISSUE TITLE
Process for requesting releases

### ISSUE THREAD

**Issue by cjmartian** [conda/conda-build#4356]
### Actual Behavior

Releases happen sporadically and there's no way (to my understanding) to request a release

### Expected Behavior

Releases either happen on a cadence or there is a formal way to request a release from the maintainers. This way users of `conda-build` can use new features as they are merged into the library.  

Community members should have a formal way of requesting a release in order to get newly added features.
Perhaps creating an issue is the way of requesting or maybe it's easier to just release once a month?

**Comment by kenodegard** [comment]
Would definitely be interested in adopting a more regular release cycle, a monthly release would be a very healthy rate. But until we've ramped up to a steady rate of changes occurring, triggering a release via a request like this is probably the best option.

**Comment by kenodegard** [comment]
It shouldn't be unreasonable for us to get a release included in our upcoming sprint starting next week. Will discuss with the rest of the team and keep you updated.

**Comment by dbast** [comment]
Given the fact that:
* Only very few conda-build code related PRs have been merged since the last release (other PRs where CI and org related)
* The merged PRs have been carefully reviewed
* There is no added feature that requires a major release with big announcement via blog/community etc.
* There are no queued important PRs to be merged soon.

it should be possible to create a release any time without huge overhat.

... it would be nice to just run rever and tag a release (instead of waiting for a sprint where that is handled for major releases coordination, which is not required here for the next minor release)

Any reasons to not proceed like that for 3.21.8 (future releases can be discussed again)?

Connor and myself have done releases of conda/* project, so we both volunteer to do it.

**Comment by jezdez** [comment]
@dbast We've briefly discussed this in the team refinement meeting and think there are a number of reasonable points here:

- you shouldn't have to wait on a release if there is only a trivial number of changes
- we're happy that you're volunteering to contribute and do releases (especially if you have prior experience)
- the future of maintaining projects under the conda organization is multi-stakeholder and @dbast and @cjmartian are stakeholders
- Anaconda internal sprints should not block release cycles, but can be the fallback if nobody else gets to the release first
- public tickets and the triaging board here on GitHub are provided not just for Anaconda internal teams, but for all contributors

All of this is of course currently a little hand-wavy because we're in a transition of the conda org on many levels. But I want to call out that I appreciate you opening this ticket here to allow the discussion to happen.

As with any other public facing project, we should make sure to follow best practices like peer review, backward compatibility and a meaningful consideration of technical debt when adding new features.

Practically speaking, I think you should be able to do the release on GitHub (release prep and then running rever etc) and prepare the updates to the conda-build feedstock for the defaults channel.

**Comment by dbast** [comment]
Thanks for the clarification!

**Comment by kenodegard** [comment]
Since we are working on https://github.com/conda-incubator/ceps/pull/26 we will close this for now unless there is community resistance to a time based release cadence.
